docs/.vuepress/components/CardList.vue

```diff
@@ -65,9 +65,11 @@ export default {
   grid-template-columns repeat(2, 1fr)
 .col-3
   grid-template-columns repeat(3, 1fr)
-.col-5
- grid-template-columns repeat(4, 1fr)
 .col-4
+ grid-template-columns repeat(4, 1fr)
+.col-5
+  grid-template-columns repeat(5, 1fr)
+.col-6
```

Comment:
This looks to apply the same css as `.col-5`?

Reply:
Should it be `grid-template-columns repeat(6, 1fr)`?

Reply:
Yep! Ty 🙌